saleor/graphql/core/utils/__init__.py

```diff
@@ -33,15 +35,48 @@ def str_to_enum(name):
     return name.replace(" ", "_").replace("-", "_").upper()
 
 
-def validate_image_file(file, field_name):
+def validate_image_file(file, field_name, error_class):
     """Validate if the file is an image."""
     if not file:
         raise ValidationError(
-            {field_name: ValidationError("File is required", code="required")}
+            {
+                field_name: ValidationError(
+                    "File is required.", code=error_class.REQUIRED
+                )
+            }
         )
     if not file.content_type.startswith("image/"):
         raise ValidationError(
-            {field_name: ValidationError("Invalid file type", code="invalid")}
+            {
+                field_name: ValidationError(
+                    "Invalid file type.", code=error_class.INVALID
+                )
+            }
+        )
+    _validate_image_format(file, field_name, error_class)
+
+
+def _validate_image_format(file, field_name, error_class):
+    """Validate image file format."""
+    Image.init()
```

Comment:
Can we move `Image.init()` outside of the function, to run it when the server starts and not for each function call? These formats are not likely to change often and loading this library probably takes some time.